### ISSUE TITLE
Oauth failing

### ISSUE THREAD

**Issue by Kxa422** [forcedotcom/cinnamon#57]
Each time I try to connect an instance, the following error appear just after clicking on Allow

https://oauth-redirector-developer-edition.na10.force.com/ is down for maintenance 

Sorry for the inconvenience. We'll be back shortly. Please email us if you need to get in touch.

**Comment by renukasingh25** [comment]
Hi .. Did u find how to resolve this issue?

I am also facing this issue in my org when i try to connect any org to the setting page

**Comment by Kxa422** [comment]
No, have send emails, no answers, I think that the project is dead.

**Comment by renukasingh25** [comment]
then how u do it then?  which automation testing tool you are using now?
Any other information will be much appreciated.

**Comment by Kxa422** [comment]
I'm using selenium with jenkins to run the tests.

**Comment by renukasingh25** [comment]
Oh. can you help me for doing that same?  I also need to have automation testing for my salesforce project.

**Comment by rejochandran** [comment]
I have a similar issue on OAuth too
the OAuth screen prompts "URL no longer exists"

Any idea how we can resolve this ?

**Comment by dudhalevikrant** [comment]
I am also getting same issue.
"URL no longer exist" while connecting org under test set up.

![urlissue](https://user-images.githubusercontent.com/27685398/31215472-3d6268ec-a9cd-11e7-862b-3483c3b7b558.png)


Anyone has an idea how to fix it?
